### PR TITLE
QEMU: open terminal only when soc_term is not launched

### PR DESCRIPTION
**Pull request by m943040028**
Signed-off-by: SY Chiu <sy.chiu@linaro.org>
Tested-by: SY Chiu <sy.chiu@linaro.org> (QEMU)

### CODE REVIEW

File: qemu.mk [hunk 1]
```diff
@@ -191,6 +191,11 @@ define run-help
 	@echo xtest 2001
 endef
 
+define launch-terminal
+	@nc -z  127.0.0.1 $(1) || \
```
Comment (jbech-linaro) [inline qemu.mk]: I like this, but can you please also update the [dependency-list](https://github.com/OP-TEE/optee_os#431-prerequisites) in optee_os to include netcat also?

**Comment by jbech-linaro** [comment]
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

File: qemu.mk (hunk 1 above)
Comment (m943040028) [inline qemu.mk]: No problem

**Comment by pascal-brand-st-dev** [comment]
Reviewed-by: Pascal Brand <pascal.brand@linaro.org>

**Comment by pascal-brand-st** [comment]
Manually merged

**Pull request closed without merging**